agent/src/agent/cli/run_test_pipeline.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import click
+import sdc_client
+
+from jsonschema import ValidationError
+from agent import destination, pipeline, source
+from agent.cli.pipeline import check_prerequisites, get_info
+from agent.modules import constants
+from agent.modules.logger import get_logger
+
+logger = get_logger(__name__, stdout=True)
+
+
+@click.command(name='run-test-pipeline')
+def run_test_pipeline():
+    """
+    Creates temporary source and pipeline to check the destination is accessible.
+    """
+    try:
+        check_prerequisites()
+        _add_source()
+        _add_pipeline()
+        _run_pipeline()
+    except click.ClickException as e:
+        click.secho(f'Test run failed: {e}', fg='red')
```

Comment:
if you want to make sure your command ran successfully you can call `exit(1)` here and your test will check if the exit code is 0